<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>753</width>
      <height>706</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>设置</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>登录设置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>账号</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>密码</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>服务器</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="Box_server_2"/>
           </item>
           <item row="0" column="1">
            <widget class="QTextEdit" name="textEdit_account_2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTextEdit" name="textEdit_password_2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_login">
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>731</width>
        <height>481</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>日志</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QTextEdit" name="log_text"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>发送失败</string>
       </attribute>
       <widget class="QTextBrowser" name="textBrowser_sendfail">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>701</width>
          <height>431</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>650</y>
        <width>731</width>
        <height>21</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_check">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>校检测试</string>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>341</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_6">
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>路径</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextBrowser" name="directory_text">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="Button_chose">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>30</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>文件选择</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_send_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>130</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>开始发送</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>邮件</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>邮件编辑</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <layout class="QFormLayout" name="formLayout_7">
             <property name="verticalSpacing">
              <number>6</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>标题</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QTextEdit" name="textEdit_title">
               <property name="maximumSize">
                <size>
                 <width>700</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>内容</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEdit_gmail">
               <property name="minimumSize">
                <size>
                 <width>400</width>
                 <height>580</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>myExitButtonFuc()</slot>
 </slots>
</ui>
